src/main/java/org/tron/common/runtime/vm/program/Program.java

```diff
@@ -1117,8 +1117,8 @@ public static String stringifyMultiline(byte[] code) {
 
   static class ByteCodeIterator {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Reduce this class from 32 to the maximum allowed 25 or externalize it in a public class. ([squid:S2972](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2972))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1633&issues=AWak_C1mdPPRVdwyhXDP&open=AWak_C1mdPPRVdwyhXDP)

[ISSUE_KEY]: AWak_C1mdPPRVdwyhXDP